### PR TITLE
Magistrate Things and Representative Navy Suit

### PR DESCRIPTION
**Pull request by Quantum-M**
**What does this PR do:**
Gives the Magistrate a locker for themselves and a suit, a new wig, and specific stamp, along with placing a existing Nanotrasen Navy Suit (for representatives) in the NT Rep's locker. Also edits the back part of the powdered wig.

**Note:** Locker not currently mapped, there are two other mapping PRs at the moment and I rather avoid mapping conflicts.

**Images of sprite/map changes (IF APPLICABLE):**
magistratelocker

Items in the locker: White gloves, white powdered wig, judge robes, executive suit, Nanotrasen Navy Suit (for Magistrates), the laceup shoes, gavel and hammer, manual to faxes, a flash, a secure briefcase, a pair of sunglasses, and (not pictured with the locker stuff) a new wig.

magistrate wig
 
Pictured to the left is the newly added wig, meant to make it look like one of those stereotypical looking cascade wig things. To the right are the edits to the back part of the powdered wig.

magistrate stamp

Magistrate stamp, paper icon, and stamp icon.

**Changelog:**

:cl: Quantum-M
add: Magistrate Locker to the code.
add: Magistrate version of the Nanotrasen Navy Suit.
add: Gives the Magistrate a new wig, called Justice wig.
add: Magistrate specific stamp.
tweak: Adds the NT Rep version of the Nanotrasen Navy Suit (which already existed) to the NT Rep locker.
/:cl:

### CODE REVIEW

**Comment by Quantum-M** [comment]
Powdered wig, @Superhats.

I could add a hat (a reasonable one) or make one, but I'll need suggestions.

**Comment by Quantum-M** [comment]
Added a new wig and stamp for the magistrate. New wig appears in the coded locker.

File: code/modules/client/asset_cache.dm [hunk 1]
```diff
@@ -191,6 +191,7 @@ You can set verify to TRUE if you want send() to sleep until the client has the
 		"large_stamp-cent.png"      = 'icons/paper_icons/large_stamp-cent.png',
 		"large_stamp-syndicate.png" = 'icons/paper_icons/large_stamp-syndicate.png',
 		"large_stamp-rep.png"	    = 'icons/paper_icons/large_stamp-rep.png',
+		"large_stamp-magistrate.png"= 'icons/paper_icons/large_stamp-magistrate.png',
```
Comment (KasparoVy) [inline code/modules/client/asset_cache.dm]: No spess behind `=`? :thinking:

**Comment by KasparoVy** [comment]
The Devil(TM) is in the details

File: code/modules/client/asset_cache.dm (hunk 1 above)
Comment (Quantum-M) [reply code/modules/client/asset_cache.dm]: @KasparoVy the intent was to keep the equal sign there aligned with the other ones.

**Comment by Quantum-M** [comment]
Conflicts fixed, I think.

**Pull request merged**